### ISSUE TITLE
Specify the sound for a specific notification

### ISSUE THREAD

**Issue by variadico** [variadico/noti#17]
> After playing with v2.0.0 for a bit, I was curious, how easy would it be to bring back the -s tag for sounds rather than speech? I like the speech option but it is much more beneficial to me to be able to specify the sound for a specific notification.
> 
> For example, I'm using noti to notify me when build/deployments are complete or have failed. It was nice being able to specify the "Hero" sound for the error notification so I could easily tell the difference. The only option I have at the moment is to set a system variable before and after the noti call, which is not ideal.

@KingKrimmson

**Comment by variadico** [comment]
To set the sound for a notification like `-s`, you have two options:
1. Set the default sound to Hero with `export NOTI_SOUND=hero`.
2. Use hero for only 1 run with `NOTI_SOUND=hero noti foo`.

However, did you also mean you wanted a different sound when a process succeeds vs fails?

**Comment by KingKrimmson** [comment]
> However, did you also mean you wanted a different sound when a process succeeds vs fails?
Yes, in my aliases, I have something like:
`"run_script="~/scripts/my_script.sh && noti -t $PSNAME -m 'Success' || noti -s Hero -t $PSNAME -m 'Failure'"`
That way there is a clearly different sound for each. Perhaps if I use your second option like:
`"run_script="~/scripts/my_script.sh && noti -t $PSNAME -m 'Success' || NOTI_SOUND=Hero noti -t $PSNAME -m 'Failure'"`
I can get my preferred functionality. My assumption was after this run, the system variable would then be set to Hero, making all other noti's set to Hero as well, but I never got around to testing it.

**Comment by variadico** [comment]
Ah, okay. I actually liked having two different sounds for the notifications as well. With the refactor, I accidentally removed the ability to do that.

What do you think about [these changes](https://github.com/variadico/noti/blob/dev/CHANGELOG.md)? 

Basically you can simplify the commands like this.
```bash
export NOTI_SOUND=Ping
export NOTI_SOUND_FAIL=Hero
# Or define that in your .zshrc file to make this permanent.

run_script="~/scripts/my_script.sh && noti -t $PSNAME -m 'Success' || noti -t $PSNAME -m 'Failure'"

# Or maybe even just
run_script="noti ~/scripts/my_script.sh"
# If my_script.sh fails, the title will automatically be "my_script.sh failed" and 
# the message "exit status 1".
```

**Comment by KingKrimmson** [comment]
That's perfect. Creates exactly the functionality I need and still allows users to set default sounds, I like it.

**Comment by variadico** [comment]
Closing issue because this has been implemented on `dev`. Though, it won't be officially released until v2.1.0.
